src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherAction.java

```diff
@@ -54,6 +55,14 @@ public ActionForward route(ActionMapping mapping, ActionForm form, HttpServletRe
         return forward;
     }
     
+    @Override
+    public ActionForward copy(ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response) throws Exception {
+    	RecurringDisbursementVoucherForm recurringForm = (RecurringDisbursementVoucherForm) form;
+    	recurringForm.getPdpStatuses().clear();
+    	ActionForward forward = super.copy(mapping, form, request, response);
```

Comment:
should the getting of the form and clearing the arrayList come after super()?

I think we also want to loop through recurring Details, and clear the DV document number from them.